lang/cs/Org.Apache.REEF.Driver/Bridge/DriverBridge.cs

```diff
@@ -141,6 +145,7 @@ internal sealed class DriverBridge
             [Parameter(Value = typeof(DriverBridgeConfigurationOptions.TraceListenersSet))] ISet<TraceListener> traceListeners,
             [Parameter(Value = typeof(EvaluatorConfigurationProviders))] ISet<IConfigurationProvider> configurationProviders,
             [Parameter(Value = typeof(DriverBridgeConfigurationOptions.TraceLevel))] string traceLevel,
+            IDriverReconnectionConfigurationProvider driverReconnectionConfigurationProvider,
```

Comment:
Because the bind is conditional -- if the user already bound `IDriverConnection`, we don't add it to the set of `EvaluatorConfigurationProviders`. If the user doesn't, we do. It is conditional to avoid Tang conflicts.